teams/teams-ps/teams/Set-TeamTargetingHierarchy.md

```diff
@@ -0,0 +1,66 @@
+---
+external help file: Microsoft.Open.Teams.CommonLibrary.dll-Help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/powershell/module/teams/connect-microsoftteams
+schema: 2.0.0
+author: mbergamo
+ms.author: mbergamo
+ms.reviewer:
+---
+# Set-TeamTargetingHierarchy
+
+## SYNOPSIS
+
+Upload a hierarchy to the tenant. A tenant may only have 1 active hierarchy. Each Set-TeamTargetingHierarchy cmdlet call will overwrite the previous one.
+
+## SYNTAX
+
+### Get (Default)
+```
+Set-TeamTargetingHierarchyStatus [-FilePath <String>]
+```
+
+## DESCRIPTION
+
+A sample CSV file uses the following format:
+
+> TargetName,ParentName,TeamId,Location:Clothing,Location:Jewelry,#Bucket1,#Bucket2 <br> Apogee,,A8A6626F-87B3-4B8A-9469-47BCD1174673,0,0 <br> New Jersey,Apogee,06763207-4F56-4DFE-96AE-4C7F9ADCFB43,0,0 <br> Basking Ridge,NewJersey,5F44CC65-9521-4F7F-9099-64320153CA07,1,0 <br> Mountain Lakes,NewJersey,58A21379-8E4D-4DA5-AA35-9CC188D8A998,0,1
+
+Based on the CSV file, the following hierarchy is created:
+
+> - Apogee <br>  - New Jersey <br>    - Basking Ridge <br>    - Moutain Lakes
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Set-TeamTargetingHierarchy -FilePath d:\hier.csv
+
+Key       Value
```

Comment:
Output from a cmdlet should be encased in ```output:

\```powershell
\> Cmdlet
\```

\```output
Result
\```